readthedocs/core/utils/__init__.py

```diff
@@ -274,6 +275,24 @@ def cancel_build(build):
     app.control.revoke(build.task_id, signal=signal.SIGINT, terminate=terminate)
 
 
+def send_email_from_object(email: EmailMultiAlternatives | EmailMessage):
+    """Given an email object, send it using our send_email_task task."""
+    from readthedocs.core.tasks import send_email_task
+
+    html_content = None
+    if isinstance(email, EmailMultiAlternatives):
+        for content, mimetype in email.alternatives:
+            if mimetype == "text/html":
+                html_content = content
```

Comment:
It seems we are missing a break here

```suggestion
                html_content = content
                break
```